testData/preferences/daniel.json5

```diff
@@ -0,0 +1,26 @@
+// # Daniel.json5
+//
+// This preference sets the desktop background to a solid color.
+//
+// ## Testing
+//
+// Windows desktop background should be replace by a green image.
+//
+{
+    "flat": {
+        "name": "Daniel",
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/applications/com.microsoft.windows.desktopBackground": {
+                        "BackgroundTypeVal": "SolidColor",
+                        "InputVal": {
+                            "Color": 1293123
```

Comment:
It may be nicer to use an array/object containing the separate RGB values. (they can then be merged using `r | g << 8 | b << 16`)

Reply:
I followed the suggestion, this is now changed to receive and object of kind: { r: 0..0xff, g: 0..0xff, b: 0..0xff }.